recipe/meta.yaml

```diff
@@ -1,40 +1,44 @@
 {% set version = "2.1.0" %}
 
 package:
-    name: expat
-    version: {{ version }}
+  name: expat
+  version: {{ version }}
 
 source:
-    fn: expat-{{ version }}.tar.bz2
-    url: http://downloads.sourceforge.net/project/expat/expat/{{ version }}/expat-{{ version }}.tar.gz
-    md5: dd7dab7a5fea97d2a6a43f511449b7cd
+  fn: expat-{{ version }}.tar.bz2
+  url: http://downloads.sourceforge.net/project/expat/expat/{{ version }}/expat-{{ version }}.tar.gz
+  md5: dd7dab7a5fea97d2a6a43f511449b7cd
 
 build:
-    number: 2
-    features:
-        - vc9  # [win and py27]
-        - vc10  # [win and py34]
-        - vc14  # [win and py35]
+  number: 3
+  features:
+      - vc9  # [win and py27]
+      - vc10  # [win and py34]
+      - vc14  # [win and py35]
```

Comment:
None of this works without Python. We don't have `vc` matrix support yet either.

xref: https://github.com/SciTools/conda-build-all/pull/36
xref: https://github.com/SciTools/conda-build-all/issues/65
xref: https://github.com/conda/conda-build/issues/1445

cc @pelson